crowbar_framework/app/models/proposal.rb

```diff
@@ -0,0 +1,118 @@
+class Proposal < ActiveRecord::Base
+  include Crowbar::ProposalMethods
+
+  # FIXME: remove this
+  class_attribute :chef_type
+
+  self.chef_type = "data_bag_item"
+
+  # FIXME: add proper i18n to errors
```

Comment:
> Yes, please :-) We should do that everywhere :-)

I have that patch ready, I just did not want to add more patches until
the current ones are reviewed :)